countries.go

```diff
@@ -0,0 +1,66 @@
+package countries
+
+import (
+	"fmt"
+	"io/ioutil"
+	"sort"
+
+	"gopkg.in/yaml.v3"
+)
+
+// GetCountry returns the country data
+func GetCountry(countryCode string) interface{} {
+	path := "lib/data/countries.yaml"
```

Comment:
This path will not be available at run time. you might need to make use of [embed](https://pkg.go.dev/embed) to get this to work as expected

Reply:
good catch! replaced it by embed